app/src/main/java/ar/com/wolox/android/training/ui/login/LoginPresenter.kt

```diff
@@ -16,13 +16,16 @@ class LoginPresenter @Inject constructor(private val sharedPreferences: SharedPr
         private const val userEmailKey = "UserEmail"
     }
 
+<<<<<<< HEAD
```

Comment:
You forget to remove this from the conflicts

Reply:
Solved in next PR update.